api/envoy/extensions/filters/network/tcp_proxy/v3/tcp_proxy.proto

```diff
@@ -247,6 +248,11 @@ message TcpProxy {
   google.protobuf.Duration max_downstream_connection_duration = 13
       [(validate.rules).duration = {gte {nanos: 1000000}}];
 
+  // Percentage-based jitter for max_downstream_connection_duration. The jitter will be added
+  // randomly to the max_downstream_connection_duration.
+  // This field is ignored if max_downstream_connection_duration is not set.
```

Comment:
OOC what are the tradeoffs of ignoring the field vs. rejecting the config (invalid configuration)?